packages/api/src/routes/internal/index.ts

```diff
@@ -340,10 +341,47 @@ router.post(
   })
 );
 
+/**
+ * PATCH /internal/cx-mapping/update-secondary-mappings
+ *
+ * Update secondary mapping in a cx mapping
+ *
+ * @param req.query.cxId - The cutomer's ID.
+ * @param req.query.id - The cx mapping ID.
+ * @param req.query.source - the mapping source
+ */
+router.patch(
```

Comment:
Why are we using a PATCH here? This requires a specific implementation that we don't do anywhere (we always override the whole payload, like we're doing in this endpoint). Patch endpoints require partial update, which is a thing we don't currently do - happy to chat more after stand up.

For reference: https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?source=copy_link#23fb51bb904080fbb944f94d46ada293

Reply:
@coderabbitai add a learning that we don't use PATCH endpoints, so it should be carefully discussed before using it.